Netkan/Sources/Curse/CurseMod.cs

```diff
@@ -69,19 +71,21 @@ public string GetName()
 
         public override string ToString()
         {
-            //return string.Format("{0}", name);
             return string.Format("{0}", title);
         }
 
-        public static CurseMod FromJson(int modId, string json)
+        public static CurseMod FromJson(string json)
         {
             CurseMod mod = JsonConvert.DeserializeObject<CurseMod>(json);
-            mod.ModId = modId;
-            foreach (CurseFile file in mod.files.Values)
-            {
-                file.Mod = mod;
-            }
```

Comment:
Why do you delete all this? This works fine if you delete `.Values`

Reply:
It creates circular references (mod -> file, file -> mod) that never get explicitly cleaned up and therefore probably causes garbage leakage. And it's not used for anything.